.github/ISSUE_TEMPLATE/ixd_user_story.md

```diff
@@ -0,0 +1,35 @@
+---
+name: 🧑‍🎨 IxD User story
+about: (DEV TEAM ONLY) A small chunk of work to be done by an Interaction Designer
+title: '(Fully descriptive title)'
+labels: 'IxD,triage'
+---
+
+<!-- Ensure the title can be understood without the parent item's context, e.g. "nimble-datepicker Interaction Design" rather than just "Interaction Design" -->
+
+## 📌 Request Background
+
+<!-- A short description of the request and requester. E.g. Who is the client and how do they expect to use the component? -->
+
+## Core Requirements
```

Comment:
Do you envision we would have an IxD user story as a task linked from a top-level "new component" / "feature" issue? If so I worry about collecting requirements and prior art in both places (in the past we've often put this info in the "Problem to Solve" and "Proposed Solution" sections of the new_component feature. I think we should pick one and delete the relevant section from the other place.

Reply:
Yes - I expect this IxD user story to be linked to the top-level feature issue, but I want to be super crisp in the handoff to IxD, so I want to reiterate the key requirements here. I can see the top-level feature being a good forum for brainstorming and discussion, and would be more work to edit down to the core requirements than it is to frame them anew here.

I'll also add a non-requirements section, because it's easy for earlier discussions to create implicit requirements.

Reply:
…but I will remove the Prior Art section, as it's better discussed in the top-level feature.